src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine truck = new Truck();
 
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
```

Comment:
The messages in the `doWork` and `stopWork` methods should be informative and indicate what type of `Machine` is working or stopping. Make sure that these methods in `Excavator`, `Bulldozer`, and `Truck` classes print out messages that fulfill this requirement.